js/apps/system/_admin/aardvark/APP/frontend/js/views/replicationView.js

```diff
@@ -566,7 +566,8 @@
         $('#repl-logger-clients tbody').html('');
         _.each(clients, function (client) {
           $('#repl-logger-clients tbody').append(
-            '<tr><td>' + client.serverId + '</td>' +
+            '<tr><td>' + client.syncerId + '</td>' +
+            '<td>' + client.serverId + '</td>' +
```

Comment:
I think if we adjust it here, we also need to adjust replicationView.ejs and add the extra column there too:
https://github.com/arangodb/arangodb/blob/devel/js/apps/system/_admin/aardvark/APP/frontend/js/templates/replicationView.ejs#L206